R/bind_ard.R

```diff
@@ -7,10 +7,18 @@
 #'   ARDs to combine. Each argument can either be an ARD,
 #'   or a list of ARDs. Columns are matched by name, and any missing
 #'   columns will be filled with `NA`.
+#' @param .distinct (`logical`)\cr
+#'   logical indicating whether to remove non-distinct values from the ARD.
+#'   Duplicates are checked across grouping variables, primary variables,
+#'   context (if present), the **statistic name and the statistic value**.
+#'   Default is `FALSE`. If a statistic name is repeated and `.update=TRUE`,
+#'   the more recently added statistics will be retained, and the other(s) omitted.
 #' @param .update (`logical`)\cr
```

Comment:
maybe not worth it but would it be better to give it a name like `.distint_stats`?

Reply:
I was struggling with the name of this arg. I wanted it to be short, but was worried it's not descriptive enough.  i think we can keep if for now, since the messaging for the arg is pretty clear. 🤞🏼